source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -299,6 +299,10 @@ package:
        field to BSON. You can include a discriminator to serialize POJOs
        nested within the field.
 
+   * - ``BsonExtraElements``
+     - Specifies any extra elements in a ``ChangeStreamDocument`` that
```

Comment:
BsonExtraElements is not specific to ChangeStreamDocument.  It just happened to be a good use case for it.  It can be applied to any POJO, just like all of the above annotations.